streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/StreamStreamJoinNode.java

```diff
@@ -198,16 +158,6 @@ public StreamStreamJoinNodeBuilder<K, V1, V2, VR> withOuterJoinWindowStoreBuilde
             return this;
         }
 
-        public StreamStreamJoinNodeBuilder<K, V1, V2, VR> withJoined(final Joined<K, V1, V2> joined) {
```

Comment:
ah, meant to leave a comment about this. no, was not called or used before this (the Joined was never invoked, only used for logging, but since we didn't ever set it, nothing was being logged to begin with 🤷‍♀️ )